app/dashboard/tokens.py

```diff
@@ -21,8 +21,6 @@
     {"addr": "0x2941deaad71adb02b944bd38ebce2f1f4c9a62dc", "name": "COLO", "decimals": 18},
     # from https://github.com/etherdelta/etherdelta.github.io/blob/master/config/main.json
     {"addr": "0x0000000000000000000000000000000000000000", "name": "ETH", "decimals": 18},
-    # for backwards compat with old data, also accept 0x0 as ETH token address
-    {"addr": "0x0",                                        "name": "ETH", "decimals": 18},
```

Comment:
any reason you removed this line?

Reply:
Sorry for this, no that was a mistake. Should be fine now.